docs/server/source/production-deployment-template-tendermint/workflow.rst

```diff
@@ -0,0 +1,188 @@
+
+The BigchainDB APIs (HTTP API and WebSocket API) should be served using TLS,
+so the organization running the cluster
+should choose an FQDN for their API (e.g. api.organization-x.com),
+register the domain name,
+and buy an SSL/TLS certificate for the FQDN.
+
+.. _things-each-node-operator-must-do-tmt:
+
+Things Each Node Operator Must Do
+---------------------------------
+
+Use a standard and unique naming convention for all instances.
+
+☐ Name of the MongoDB instance (``mdb-instance-*``)
+
+☐ Name of the BigchainDB instance (``bdb-instance-*``)
+
+☐ Name of the NGINX instance (``ngx-http-instance-*`` or ``ngx-https-instance-*``)
+
+☐ Name of the OpenResty instance (``openresty-instance-*``)
+
+☐ Name of the MongoDB monitoring agent instance (``mdb-mon-instance-*``)
+
+☐ Name of the Tendermint instance (``tendermint-instance-*``)
+
+Example
+^^^^^^^
+
+.. code:: text
+
+  {
+    "MongoDB": [
+      "mdb-instance-1",
+      "mdb-instance-2",
+      "mdb-instance-3",
+      "mdb-instance-4"
+    ],
+    "BigchainDB": [
+      "bdb-instance-1",
+      "bdb-instance-2",
+      "bdb-instance-3",
+      "bdb-instance-4"
+    ],
+    "NGINX": [
+      "ngx-instance-1",
+      "ngx-instance-2",
+      "ngx-instance-3",
+      "ngx-instance-4"
+    ],
+    "OpenResty": [
+      "openresty-instance-1",
+      "openresty-instance-2",
+      "openresty-instance-3",
+      "openresty-instance-4"
+    ],
+    "MongoDB_Monitoring_Agent": [
+      "mdb-mon-instance-1",
+      "mdb-mon-instance-2",
+      "mdb-mon-instance-3",
+      "mdb-mon-instance-4"
+    ],
+    "Tendermint": [
+      "tendermint-instance-1",
+      "tendermint-instance-2",
+      "tendermint-instance-3",
+      "tendermint-instance-4"
+    ]
+  }
+
+
+☐ Generate three keys and corresponding certificate signing requests (CSRs):
+
+#. Server Certificate for the MongoDB instance
+#. Client Certificate for BigchainDB Server to identify itself to MongoDB
+#. Client Certificate for MongoDB Monitoring Agent to identify itself to MongoDB
+
+Use the self-signed CA to sign those three CSRs:
+
+* Three certificates (one for each CSR).
+
+For help, see the pages:
+
+* :doc:`How to Generate a Server Certificate for MongoDB <../production-deployment-template/server-tls-certificate>`
+* :doc:`How to Generate a Client Certificate for MongoDB <../production-deployment-template/client-tls-certificate>`
+
+☐ Make up an FQDN for your BigchainDB node (e.g. ``mynode.mycorp.com``).
+Make sure you've registered the associated domain name (e.g. ``mycorp.com``),
+and have an SSL certificate for the FQDN.
+(You can get an SSL certificate from any SSL certificate provider.)
+
+☐ Ask the managing organization for the user name to use for authenticating to
+MongoDB.
+
+☐ If the cluster uses 3scale for API authentication, monitoring and billing,
+you must ask the managing organization for all relevant 3scale credentials -
+secret token, service ID, version header and API service token.
```

Comment:
Is this list of 3scale credentials still accurate (with the new 3scale)?

Reply:
I will keep it the same for now, i think @shahbazn can address this in his PR for apicast deployment. I am not entirely sure which parameters are needed.

Reply:
:+1: